testing/kfctl/kfctl_go_test.py

```diff
@@ -99,21 +109,22 @@ def test_build_kfctl_go(app_path, project, use_basic_auth, use_istio):
   # We don't run with retries because if kfctl init exits with an error
   # but creates app.yaml then rerunning init will fail because app.yaml
   # already exists. So retrying ends up masking the original error message
+  config_file = "kfctl_gcp_iap_master.yaml"
+  if use_basic_auth:
+    config_file = "kfctl_gcp_basic_auth_master.yaml"
+  with open(os.path.join(src_dir, "bootstrap/config", config_file), 'r') as f:
```

Comment:
Why not just take the full URL of the config file as an argument of kfctl_go_test.py?

kfctl supports https://github.com/hashicorp/go-getter syntax.
So we can specify a full absolute local path or http location.

Reply:
done

Reply:
It still doesn't look like your taking the full URL.
The repos will be checked out into /mnt/src/....
So my expectation was that you kfctl_go_test.py would just take the full path of the config_file
e.g.
kfctl_go_test.py --config=/mnt/src/kubeflow/kubeflow/bootstrap/config/kfctl_gcp_basic_auth_master.yaml